doc/glossary.md

```diff
@@ -613,9 +613,77 @@ directory to all files contained within the directory.
 <br />
 
 <dl>
-<dt></dt><br />
+<section id="mount"><dt>
+
+#### <dfn><code>mount</code></dfn>&nbsp;<a role="button" aria-expanded="false" tabindex="0" href="#mount" id="mount-ref">⚓</a>
+
+</dt>
+<div align="right"><note place="source"><ref target="chap11">
+
+[⑪📑][]
+
+</ref></note></div>
 <dd>
 
+To temporarily mount a filesystem to the Linux virtual directory, use the
+`mount` command. The basic format for the `mount` command is:
+
+```console
+$ mount -t fstype device mountpoint
+```
+
+Use the `-t` command-line option to specify the filesystem type of the device:
+
+```console
+$ sudo mount -t ext4 /dev/sdb1 /media/usb1
+```
+
+If you specify the `mount` command with no parameters, it displays all of the
+devices currently mounted on the Linux system. Be prepared for a long output
+though, as most Linux distributions mount many virtual devices in the virtual
+directory to provide information about system resources. Listing 11.1 shows a
+partial output from a `mount` command.
+
+```console
+$ mount
+...
+/dev/sda2 on / type ext4 (rw,relatime,errors=remount-ro,data=ordered)
+/dev/sda1 on /boot/efi type vfat
+ (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=iso8859
+-1,shortname=mixed,errors=remount-ro)
+...
+/dev/sdb1 on /media/usb1 type ext4 (rw,relatime,data=ordered)
+/dev/sdb2 on /media/usb2 type ext4 (rw,relatime,data=ordered)
+rich@rich-TestBox2:~$
+```
+
+**Listing 11.1: Output from the `mount` command**
```

Comment:
~~~suggestion


<div align="center"><br /><br />
<figure>

```console
$ mount
...
/dev/sda2 on / type ext4 (rw,relatime,errors=remount-ro,data=ordered)
/dev/sda1 on /boot/efi type vfat
 (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=iso8859
-1,shortname=mixed,errors=remount-ro)
...
/dev/sdb1 on /media/usb1 type ext4 (rw,relatime,data=ordered)
/dev/sdb2 on /media/usb2 type ext4 (rw,relatime,data=ordered)
rich@rich-TestBox2:~$
```
<br />
<figcaption>

**Listing 11.1: Output from the `mount` command**

</figcaption>
</figure>
</div><br /><br />

~~~